package/cloudshell/cm/customscript/domain/script_downloader.py

```diff
@@ -43,10 +44,16 @@ def download(self, url, auth):
         :rtype: ScriptsData
         """
         # identify download strategy
-        if auth.username == 'GITLAB':
-            # GitLab strategy
-            scripts_data = GitLabScriptDownloader(self.logger, ALLOWED_FILES_PATTERN, self.cancel_sampler)\
-                .download(url, auth)
+        if auth.username in ['GITLAB','GITHUB']:
+            if auth.username == 'GITLAB':
+                # GitLab strategy
+                scripts_data = GitLabScriptDownloader(self.logger, ALLOWED_FILES_PATTERN, self.cancel_sampler)\
+                    .download(url, auth)
+
+            if auth.username == 'GITHUB':
+                # GitLab strategy
+                scripts_data = GitHubScriptDownloader(self.logger, ALLOWED_FILES_PATTERN, self.cancel_sampler) \
+                    .download(url, auth)
```

Comment:
Why not add a simple "elif" like:

"elsif auth.username == 'GITHUB':"  ?

Reply:
yes indeed